sonar-ws/src/main/protobuf/ws-issues.proto

```diff
@@ -181,3 +181,24 @@ message Component {
   optional int64 projectId = 9;
   optional int64 subProjectId = 10;
 }
+
+// Response of GET api/issues/changelog
+message ChangelogWsResponse {
+  repeated Changelog changelog = 1;
+
+  message Changelog {
+    optional string user = 1;
```

Comment:
It was like that in rails. I can deprecate it and add a login field if this is the way we do now.